frontend/src/features/admin-form/responses/FeedbackPage/issue/IssueTable.tsx

```diff
@@ -0,0 +1,39 @@
+import { Column } from 'react-table'
+
+import { DateCell } from '~features/admin-form/responses/FeedbackPage/DateCell'
+
+export const ISSUE_TABLE_COLUMNS: Column[] = [
+  {
+    Header: '#',
+    accessor: (_row, i) => i + 1,
+    sortType: 'number',
+    minWidth: 50, // minWidth is only used as a limit for resizing
+    width: 50, // width is used for both the flex-basis and flex-grow
+    maxWidth: 100, // maxWidth is only used as a limit for resizing
+  },
+  {
+    Header: 'Date',
+    accessor: 'timestamp',
+    sortType: 'number',
+    Cell: DateCell,
+    minWidth: 80, // minWidth is only used as a limit for resizing
+    width: 80, // width is used for both the flex-basis and flex-grow
+    maxWidth: 120, // maxWidth is only used as a limit for resizing
+  },
```

Comment:
nit: can consider extracting these since they're also used in `ReviewTable`

Reply:
not fussed! can leave as it is

Reply:
alright, thanks!